server/terraform/README.md

```diff
@@ -182,11 +183,13 @@ emacs main.tf
 
 terraform init
 terraform apply
-# Should create all resources without any errors
+# THIS MAY THROW AN ERROR:
+#   Error: googleapi: Error 409: Sink _Default already exists, alreadyExists
```

Comment:
yeah terraform is strange. i should dig up my notes on this for dev.

how does logging work in firestore anyway? if we remove app engine and cloud storage, what logs do we even see?

Reply:
We'll get logs from the Cloud Functions, those output logs roughly the same way that AppEngine does. Firestore will log deploys (changes to the Rules), but not access requests.